### ISSUE TITLE
Feature request: sort installed formulae by size

### ISSUE THREAD

**Issue by cfvescovo** [brunophilipe/Cakebrew#244]
This is an awesome project! It would be even more convenient if the user could sort installed formulae by size, therefore making much easier to free up disk space.

**Comment by ssbarnea** [comment]
Indeed, size and maybe even access time if available could be a very useful information for cleaning the unused leaves.
